fire/SFMainMod.F90

```diff
@@ -916,21 +922,53 @@ subroutine  crown_damage ( currentSite )
 
           do while(associated(currentCohort))  
              currentCohort%fraction_crown_burned = 0.0_r8
+             currentCohort%crown_FI              = 0.0_r8  !critical fire intensity for passive crown fire
+             currentCohort%crown_fire            = 0       !flag for passvie crown fire ignition
+             
              if (EDPftvarcon_inst%woody(currentCohort%pft) == 1) then !trees only
+                
+                ! Calculate clear branch bole height at base of crown
+                ! inst%crown = crown_depth_frac (PFT)
+                height_cbb = currentCohort%hite*EDPftvarcon_inst%crown(currentCohort%pft)
+                
+                ! Evaluate for passive crown fire ignition
+                if (EDPftvarcon_inst%crown_fire(currentCohort%pft) == 1) then
+                   
+                   ! Crown fuel ignition potential, EQ 8 Bessie and Johnson 1995
+                   currentCohort%crown_FI = (0.01 * height_cbb * SF_val_crown_ignition_energy)**1.5
+
+                   if (currentCohort%crown_FI >= crown_fire_threshold) then ! 200 kW/m = threshold for crown fire potential
+                      
+                      ! Initiation of passive crown fire, EQ 9 Bessie and Johnson 1995                     
+                      currentCohort%ignite_crown = currentPatch%FI/currentCohort%crown_FI
+                      if (currentCohort%ignite_crown > 1.0_r8) then
+                         currentCohort%crown_fire = 1  ! passive crown fire ignited
+                         currentCohort%fraction_crown_burned =  1.0_r8
+                      else ! evaluate crown damage based on scorch height
+                         currentCohort%fraction_crown_burned =  0.0_r8
+                      endif ! passive crown fire
+                   else
+                      currentCohort%crown_fire = 0 ! no crown fire today
+                      currentCohort%fraction_crown_burned =  0.0_r8
+                   endif ! crown fire intensity
+                else   
+                   currentCohort%crown_fire = 0 ! not crown fire plant
+                   currentCohort%fraction_crown_burned =  0.0_r8
+                endif
+                
```

Comment:
@jkshuman and everyone, I want to clarify that this way crown fire behavior is separate from scorch height. The canopy-bottom height enters the empirical formula for crown fuel ignition potential above, but scorch height does not influence the outcome. Is this ok?

Reply:
@slevisconsulting the scorch height (Patch%SH) is calculated in relation to the fire intensity (Patch%FI). 
https://github.com/jkshuman/fates/blob/f3077dfda1ceba060cb8493bf868835d8bfe7cd9/fire/SFMainMod.F90#L885-L886
currentPatch%SH = currentPatch%SH + f_ag_bmass * &
               EDPftvarcon_inst%fire_alpha_SH(currentCohort%pft) * (currentPatch%FI**0.667_r8) 

passive crown fire also uses the (Patch%FI) to evaluate if the energy of the fire is enough to ignite the crown. So, they are certainly connected.